ngi_reports/common/ign_sample_report.py

```diff
@@ -71,22 +73,27 @@ def parse_qualimap(self):
                     for line in fh:
                         line = line.strip()
 
+                        def get_after_equals(s):
```

Comment:
Much nicer, thanks! :+1: 

Can even do it [without a regex](http://stackoverflow.com/questions/12572362/get-a-string-after-a-specific-substring) (more pythonic I'm told):

```python
return my_string.split("=",1)[1] 
```